entrypoint.sh

```diff
@@ -5,6 +5,14 @@ set -e
 export MEDIA_DATABASE_HOST="${MEDIA_DATABASE_HOST:-db}"
 export MEDIA_DATABASE_PORT="${MEDIA_DATABASE_PORT:-5432}"
 export MEDIA_DATABASE_NAME="${MEDIA_DATABASE_NAME:-mediafiledata}"
+if [ -n "$MEDIA_DATABASE_USER_FILE" ];
+then
+	MEDIA_DATABASE_USER=$(cat $MEDIA_DATABASE_USER_FILE)
+fi;
+if [ -n "$MEDIA_DATABASE_PASSWORD_FILE" ];
+then
+	MEDIA_DATABASE_PASSWORD=$(cat $MEDIA_DATABASE_PASSWORD_FILE)
+fi;
 export MEDIA_DATABASE_USER="${MEDIA_DATABASE_USER:-openslides}"
 export MEDIA_DATABASE_PASSWORD="${MEDIA_DATABASE_PASSWORD:-openslides}"
 PGPASSWORD="$MEDIA_DATABASE_PASSWORD"
```

Comment:
Please try to separate these variables into a third shell-script and export them. Then, source this shell-script into the both entrypoints.

Reply:
Mmh, if we use a different file, I have a different idea: I separate all vars in a file and call them with `source` in both entrypoints.